### ISSUE TITLE
Add a video that shows how to label a DOCX template

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/AssemblyLine-docs#108]
Will need to record one separately from the bootcamp, we only plan on doing the PDF. 

Was the top checkbox in 1.3b.

**Comment by plocket** [comment]
It wasn't articulated well in the checkbox item, but part of the concern there is how quickly the videos can become out of date for the weaver, especially right now. We might be able to think about how to create videos with more longevity, even if they just say 'check the docs' and have a link in the description of the video.

**Comment by samglover** [comment]
I don't think this is necessary given the [DOCX page](https://assemblyline.suffolklitlab.org/docs/docassemble_intro/working-with-docx) and the new DOCX labeler tool.
